tensorflow/python/kernel_tests/nn_ops/rnn_cell_test.py

```diff
@@ -1354,6 +1354,62 @@ def testLSTMBlockCellErrorHandling(self):
               forget_bias=forget_bias,
               cell_clip=cell_clip,
               use_peephole=use_peephole))
+      
+  @test_util.run_in_graph_and_eager_modes
+  def testLSTMBlockCellErrorHandling2(self):
```

Comment:
Make the names more descriptive - like `testLSTMBlockCellEmptyInputRaisesError`

You can also put the inputs directly into the `gen_rnn_ops.lstm_block_cell` call, rather than defining them first and repeating them in `x=x, cs_prev=cs_prev,...`.

I'm not sure you need two tests.